n4js-libs/packages/n4js-cli/package.json

```diff
@@ -7,7 +7,7 @@
   "internalVersion": "2019-03-15T08:46:12Z",
   "description": "N4JS compiler and command line tools.",
   "scripts": {
-    "postinstall": "./npm-postinstall.js",
+    "postinstall": "node ./npm-postinstall.js",
```

Comment:
Have you tested that it still runs on macOS?

Reply:
yes, it still works on macOS